### ISSUE TITLE
Alphavantage Symbolsearch aufnehmen

### ISSUE THREAD

**Issue by m-s-b** [waoler/ioBroker.stockmarket#1]
Wenn du die Symbolsuche von Alphavantage:
https://www.alphavantage.co/documentation/#symbolsearch
mit in die Doku und / oder Adapter aufnimmst erleichterst du den Nutzern sicherlich das auffinden der passenden Akteinsymbole.

**Comment by waoler** [comment]
Dank für den Vorschlag. Hab es fürs erste in der Readme angepasst. Ich werde mir da aber noch was überlegen, damit das in Zukunft Userfreundlicher ist. Wenn du eine Idee hast, immer her damit :)

**Comment by m-s-b** [comment]
Luxus isst natürlich die Suche im Adapter direkt auszuführen und die eigenen APIkey zu verwenden. Ich glaube wer es schafft iobroker und den Adapter zu installieren kommt mit dem Link zur Suche klar. 
Spannender fände ich die Möglichkeit die Abfrage URL für jede Aktie anzupassen.
Mir persönlich wäre z.B. ein Tageswert ausreichend und nicht das 5 min Intervall.

**Comment by waoler** [comment]
Ja, denke auch erstmal reicht das. Wenn ich die Zeit finde, bringe ich da noch ein Update zu.
Die Zeiten kannst du anpassen. Dazu musst du nur die Zeitplanung anpassen. Unter "Instanzen" Expertenmodus aktivieren.
![SharedScreenshot](https://user-images.githubusercontent.com/5882862/69145132-3b702480-0acd-11ea-9031-0565b73904f8.jpg)

Ich hab es so, dass einmal am Tag um 16:15 er die Kurse holt.

**Comment by m-s-b** [comment]
Ja, das mit den Cron ist mir schon klar :) 
Dachte eher, wenn man andere Abfragen verwenden möchte...
https://www.alphavantage.co/query?function=CURRENCY_EXCHANGE_RATE&from_currency=CHF&to_currency=EUR&apikey=

**Comment by ihavenonick** [comment]
Mir würde schon reichen, wenn ich die gefundenen Symbole auch so eintragen könnte, dass der Adapter was macht.
Beispiel, für Adidas bekomme ich dieses Symbol:
symbol | "ADS.DEX"
![grafik](https://user-images.githubusercontent.com/36501310/69530536-79a89080-0f72-11ea-956a-e5833743c0bc.png)

Also trage ich im Adapter ADS.DEX und nichts passiert.

**Comment by ihavenonick** [comment]
Ok, funktioniert, dauert nur sehr lange bis hier ein Objekt angelegt wird.
Nur warum bekomme ich den Dollar Kurs, wenn ich mir schon extra einen deutschen Handelsplatz aussuche?
![grafik](https://user-images.githubusercontent.com/36501310/69540360-b1213800-0f86-11ea-9a8c-20488ddfd7c5.png)
